utils/constants.go

```diff
@@ -47,6 +47,10 @@ var (
 		ClusterAdd:      http.MethodPost,
 		ClusterRemove:   http.MethodPost,
 	}
+	//RunAsUser is the user id for mysql
+	RunAsUser int64 = 1001
```

Comment:
not used, remove it,  if you want use, please call `user.Lookup("mysql")` is more safe.